sample.json

```diff
@@ -0,0 +1,15 @@
+{
+  "uuid" : "path/to/example.tif",
```

Comment:
UUID can be derived from the dataset URI and the image URI, so this seems unnecessary.